immobilienscout.gemspec

```diff
@@ -1,17 +1,36 @@
-Gem::Specification.new do |s|
-  s.name         = 'immobilienscout'
-  s.version      = '0.0.2'
-  s.date         = '2019-03-11'
-  s.summary      = 'A ruby gem to handle Immobilienscout API'
-  s.description  = 'This gem provides HTTP endpoints to connect your app with Immobilienscout API'
-  s.authors      = ['Angela Guette', 'Wiebke Frey', 'Laura Garcia']
-  s.email        = 'backend-team@homeday.de'
-  s.files        = Dir.glob('lib/**/*') + ['README.md']
-  s.homepage     = 'https://github.com/homeday-de/immobilienscout'
-  s.license      = 'MIT'
-  s.require_path = 'lib'
-  s.add_dependency 'json'
-  s.add_dependency 'multipart-post'
-  s.add_dependency 'activesupport'
-  s.add_dependency 'webmock'
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'immobilienscout/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'immobilienscout'
+  spec.version       = Immobilienscout::VERSION
+  spec.authors       = ['Angela Guette', 'Wiebke Frey', 'Laura Garcia']
+  spec.email         = ['backend-team@homeday.de']
+
+  spec.summary       = 'A ruby gem to handle Immobilienscout API'
+  spec.description   = 'This gem provides HTTP endpoints to connect your app with Immobilienscout API'
+  spec.homepage      = 'https://github.com/homeday-de/immobilienscout'
+  spec.license       = 'MIT'
+
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://github.com/homeday-de/immobilienscout'
+  spec.metadata['changelog_uri'] = 'https://github.com/homeday-de/immobilienscout/blob/master/CHANGELOG.md'
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'json'
+  spec.add_dependency 'multipart-post'
+  spec.add_dependency 'activesupport'
+  spec.add_dependency 'webmock'
+  spec.add_development_dependency 'bundler', '~> 2.0'
+  spec.add_development_dependency 'rake', '~> 10.0'
```

Comment:
Gemfile.lock has Rake 12.x, why are we downgrading now?

Reply:
This was generated by `Bundler` 🤔 I will fix it here